uber/models.py

```diff
@@ -1151,7 +1156,7 @@ def ribbon_and_or_badge(self):
 
     @property
     def is_dealer(self):
-        return bool(self.tables and self.tables != '0' and (not self.registered or self.amount_paid or self.cost))
+        return bool(self.tables and self.tables != '0' and self.tables != '0.0' and (not self.registered or self.amount_paid or self.cost))
```

Comment:
As mentioned in Slack (but I'm writing it here for anyone who follows PRs but isn't in Slack), this is for the fairly common use-case where admins want to assign a group a half-table, i.e., 0.5 tables. MAGFest doesn't currently take advantage of this afaik (although I think we want to?).

As also mentioned on Slack, there's probably a better way of handling that use-case. Something to look at refactoring after pre-reg, I think.